pkgs/os-specific/linux/nvidia-x11/default.nix

```diff
@@ -115,14 +115,12 @@ rec {
   # Last one without the bug reported here:
   # https://bbs.archlinux.org/viewtopic.php?pid=2155426#p2155426
   legacy_535 = generic {
-    version = "535.154.05";
-    sha256_64bit = "sha256-fpUGXKprgt6SYRDxSCemGXLrEsIA6GOinp+0eGbqqJg=";
-    sha256_aarch64 = "sha256-G0/GiObf/BZMkzzET8HQjdIcvCSqB1uhsinro2HLK9k=";
-    openSha256 = "sha256-wvRdHguGLxS0mR06P5Qi++pDJBCF8pJ8hr4T8O6TJIo=";
-    settingsSha256 = "sha256-9wqoDEWY4I7weWW05F4igj1Gj9wjHsREFMztfEmqm10=";
-    persistencedSha256 = "sha256-d0Q3Lk80JqkS1B54Mahu2yY/WocOqFFbZVBh+ToGhaE=";
-
-    patches = [ rcu_patch ];
```

Comment:
Just out of curiosity, how did you decide to remove this patch? Does it simply not applying anymore?

Reply:
Patches for these drivers are pretty much all for kernel compatibility, and those fixes tend to be included with new driver releases from nvidia. So I default to just removing all patches on update and then adding them back if it breaks the build.

Reply:
> Patches for these drivers are pretty much all for kernel compatibility, and those fixes tend to be included with new driver releases from nvidia. So I default to just removing all patches on update and then adding them back if it breaks the build.

OK makes sense, but it means you have to test the build of this driver against any kernel branch in Nixpkgs now?

Reply:
https://github.com/NixOS/nixpkgs/pull/352468#issuecomment-2448543129

Yup